src/pineko/cli/fonll.py

```diff
@@ -171,4 +171,5 @@ def fonll_tcards(theoryid, cfg):
     tcard_parent_path = theory_card.path(theoryid).parent
     if "FONLL" not in tcard["FNS"]:
         raise TheoryCardError("The theorycard does not correspond to an FONLL scheme.")
-    _ = fonll.produce_fonll_tcards(tcard, tcard_parent_path, theoryid)
+    fonll.dump_tcards(tcard, tcard_parent_path, theoryid)
+    return
```

Comment:
You do not need to `return` explicitly. Empty `return` returns `None`, and the same happens if you reach the end of the function without returning.

Reply:
```suggestion
```
you're right